tools/c7n_azure/c7n_azure/resources/access_control.py

```diff
@@ -211,6 +229,48 @@ def validate(self):
             )
 
 
+@RoleAssignment.filter_registry.register('scope-is-subscription')
```

Comment:
Why not just have one scope filter and pass the scope value?

Reply:
Do you mean like?

```
- type: scope
  value: subscription

- type: scope
  value: resource-group
```

Reply:
Do you need an ```op```?

Reply:
Yeah like that - no need for op. Your custom filter can take on any schema.